src/taipy/core/data/data_node.py

```diff
@@ -549,7 +550,7 @@ def is_ready_for_reading(self) -> bool:
 
     @property  # type: ignore
     @_self_reload(_MANAGER_NAME)
-    def is_up_to_date(self) -> bool:
+    def is_valid(self) -> bool:
         """Indicate if this data node is up-to-date.
```

Comment:
The docstring here can be misleading since we have a new `is_up_to_date` property